cms/templates/widgets/header.html

```diff
@@ -67,25 +68,29 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Course"
                   <li class="nav-item nav-course-courseware-updates">
                     <a href="${course_info_url}">${_("Updates")}</a>
                   </li>
+                  % if not pages_and_resources_mfe_enabled:
                   <li class="nav-item nav-course-courseware-pages">
                     <a href="${tabs_url}">${_("Pages")}</a>
                   </li>
+                  % endif
+                  % if pages_and_resources_mfe_enabled:
```

Comment:
@AhtishamShahid is pages & resources under updates dropdown?

Reply:
Yes if the `pages_and_resources_mfe` flag is enabled it will be displayed under `updates` because `pages` will be removed from the list.

Reply:
but the change indicating to show `Pages & Resouces` under `Content` dropdown. See

Reply:
@asadazam93

Reply:
@awaisdar001 Seems correct to me. It will appear below the updates dropdown button on every studio page.